backend/preprocessing/hysteresis_loops.py

```diff
@@ -148,8 +146,9 @@ def main():
                         columns=["n_cycles", "hysteresis_area", "stiffness", "creep"]
                     )
 
-                    experiment_metadata.N_fail = 100
+                    experiment_metadata.N_fail = 100  # TODO FIXME!
```

Comment:
@sbancal Is it okay to have a TODO fixme ? (should have a link to an issue) ? or do you track it in another way ?

Reply:
The code is from Scott, I'm wondering if it is relevant to rewrite/clean it...

Reply:
@guilbep good idea, I'll reference issue #136 

@blueur yes. One day ... (?)